pyscal/pyscalcli.py

```diff
@@ -205,21 +204,7 @@ def pyscal_main(
 
     if int_param_go is not None and int_param_wo is None:
         raise ValueError("Don't use int_param_go alone, only int_param_wo")
-    if (
-        int_param_wo is not None
-        and isinstance(int_param_wo, list)
-        and len(int_param_wo) > 1
-    ) or (
-        int_param_go is not None
-        and isinstance(int_param_go, list)
-        and len(int_param_go) > 1
-    ):
-        warnings.warn(
-            "SATNUM specific interpolation parameters are deprecated in "
-            "the pyscal command line client. "
-            "Use interp_relperm from subscript or the API directly",
-            FutureWarning,
-        )
```

Comment:
This is impossible to reach, as `argparse` bails out before this.

Reply:
Noting the slight inconsistency that the description states this function is available mainly to be used _without_ argparse.. Not sure if this should be removed if it is intended to be used from other places..

Reply:
Thank you for reading the comments in the code. You are completely right.

Reply:
You are completely right, this check should be there, and tested.